backup.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+#
+# Script made for backup PostgreSQL database from local (${POSTGRES_HOST}=127.0.0.1)
+# or remote host. Created backup puts on S3 storage. On completion script calls
+# notification script hooks/00-webhook.sh which sends report to given Telegram Chat
+
+set -euo pipefail
+IFS=$'\n\t'
+
+# Will be name of directory in backet dd-mm-yyyy_hh-mm-ss
+timestamp="$(date +%F_%T)"
+
+# Export stuff
```

Comment:
It's exporting postgres password. We know it for sure.

Reply:
Removed